### ISSUE TITLE
Title Attribute for Social Media Icons

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1259]
### Descriptive summary

In Hyrax 2.0.0 alpha the social media icons to share a work or fileset use a `title` attribute on the link for the icon and is not read by default by most screen readers. Suggest using an additional `<span>` with text and hiding the SVG icon from screen readers. Or ARIA enhanced SVG.

https://www.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/

### Rationale

Users of assistive technology will not be able to use the link to share repo resources because the `title` attribute is not read by default. Add plain text in a `<span>`and hide it visually so that screen reader users can share content.

### Expected behavior

Screen reader users should be able to share content via social media.

### Actual behavior
It is indiscernible from the content as presented because the title attribute is not read and the icon is SVG without text.

### Steps to reproduce the behavior

1. Test the icons using VoiceOver or a screen reader

### Related work

https://github.com/psu-stewardship/scholarsphere/issues/514
https://github.com/psu-stewardship/scholarsphere/pull/557

**Comment by adamjarling** [comment]
@mtribone These are the right share icons, correct?

![accessibility-share-icons](https://user-images.githubusercontent.com/3020266/39429915-07f7023a-4c52-11e8-9b18-13850c48b906.png)

**Comment by mtribone** [comment]
@adamjarling Yup. They are the correct share icons.
